src/components/graph-view.js

```diff
@@ -1723,7 +1738,14 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
           deltaY: 0,
         };
 
-        this.setZoom(viewTransform.k || 1, offX, offY, 0);
+        if (isZooming) {
+          this.setDiscreteZoom(viewTransform.k * (1 + zoom), 0, [
```

Comment:
what's the difference between `setDiscreteZoom` and `setZoom`. Seems like they have a similar API and produce similar transforms 
cc @kbusemeyer

Reply:
`setDiscreteZoom` zoom on a point, vs `setZoom` translate and zoom. I would suggest to rename them if possible?

Reply:
^ yup essentially what @hverlin said, for the new flow view, we just wanted to scale for zooming and not move to specific points and it was easier to just use the built in d3 methods. 

Also, the `modifyZoom`, which calls `setZoom` does some other calculations based off of previous transforms, but we knew the explicit zoom level we wanted to be at, so the new flow view just uses the `setDiscreteZoom` method as opposed to going through `modifyZoom`.

Feel free to change the naming, though!